build-tools/automation/azure-pipelines-nightly.yaml

```diff
@@ -11,7 +11,7 @@ pr:
 # External sources, scripts, tests, and yaml template files.
 resources:
   repositories:
-  - repository: yaml
+  - repository: xamarin-templates
```

Comment:
If we are working to standardize this name across repos, should we use the repo name `yaml-templates` instead of `xamarin-templates? I wonder if the latter name might get confused with https://github.com/xamarin/xamarin-templates.